bzt/modules/xunitcsharp.py

```diff
@@ -0,0 +1,54 @@
+"""
+Copyright 2018 BlazeMeter Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+from subprocess import CalledProcessError, STDOUT
+
+from bzt import TaurusConfigError
+from bzt.modules import SubprocessedExecutor
+from bzt.engine import HavingInstallableTools
+from bzt.utils import get_full_path, RequiredTool
+
+
+class XUnitExecutor(SubprocessedExecutor):
```

Comment:
For `executor: xunit` syntax to work you have to add an entry into `modules` section of `bzt/resources/base-config.yml`.

Reply:
Updated module, but I'm not  sure the namespace is correct